src/Microsoft.DotNet.Build.Tasks.net45/Microsoft.DotNet.Build.Tasks.net45.csproj

```diff
@@ -38,6 +39,7 @@
     <TargetingPackReference Include="System.Threading.Tasks" />
     <TargetingPackReference Include="System.Xml" />
     <TargetingPackReference Include="System.Xml.Linq" />
+    <TargetingPackReference Include="System.Windows.Forms" />
```

Comment:
Yes this is only going to work on .NET Framework but it is supported when building UAP right? I tested locally for building the whole test tree in UAP and in .NET Core and it worked fine, however I can move it to be netstandard1.6 and use the metadata reader and output the resources manually without the ResxResourceWriter. 

Why I would need to use the metadata reader is because Assembly.LoadFrom is not available until NS2.0 and we are currently building against NS1.6; are we planning on moving to the new toolchain and 2.0? 

Do you want me to go with the NS approach?